internal/librarian/generate.go

```diff
@@ -112,11 +112,21 @@ func newGenerateRunner(cfg *config.Config) (*generateRunner, error) {
 	if err != nil {
 		return nil, err
 	}
-	repo, err := cloneOrOpenLanguageRepo(workRoot, cfg.Repo, cfg.CI)
+	if cfg.APISource == "" {
```

Comment:
To avoid nested if and the else I would right this code as:

```go
apiSource := cfg.APISource
if apiSource == "" {
apiSource = "https://github.com/googleapis/googleapis"
}
if _, err := cloneOrOpenRepo(workRoot, apiSource, cfg.CI); err != nil {
return nil, err
}
```